r/R/query-engine.R

```diff
@@ -259,6 +221,49 @@ ExecPlan <- R6Class("ExecPlan",
       )
     },
     Stop = function() ExecPlan_StopProducing(self)
+  ),
+  private = list(
+    .config_aggregation = function(node, data, grouped, group_vars) {
+      # Project to include just the data required for each aggregation,
```

Comment:
cc @westonpace @nealrichardson @paleolimbot I added this function in order to fix the style issue in R. But a few tests are failing and example trace is as follows.

```r
Error (test-dataset-dplyr.R:193:3): count()
Error in `collect(.)`: object of type 'builtin' is not subsettable
```

What could be the issue? I am not quite clear about the R error.

Reply:
It worked well, @paleolimbot 👍